webpack.config.js

```diff
@@ -63,7 +63,6 @@ module.exports = function(env, argv) {
       argv.mode === 'development'
         ? {
             contentBase: path.resolve(__dirname, './demo'),
-            open: true,
```

Comment:
does this mean that running `npm start` won't automatically open the demo page in the browser anymore?

Reply:
To keep this, we can use the `--open` cli option of webpack serve https://github.com/webpack/webpack-cli/blob/master/SERVE-OPTIONS.md

```json
  "scripts": {
    "demo:serve": "webpack-dev-server demo/App.jsx --mode development",
    "demo:build": "webpack demo/App.jsx --mode production",
    "start": "npm run demo:serve -- --open",
    "test:e2e": "cross-env START_SERVER_AND_TEST_INSECURE=1 start-server-and-test demo:serve https-get://localhost:9000 test:e2e",
    "test:eslint": "eslint src/"
  },
```

Reply:
And please leave `open: false` so that the intention not to open a browser is explicit.